src/ifs/wl_seat.rs

```diff
@@ -1202,7 +1212,7 @@ impl WlSeat {
     fn send_capabilities(self: &Rc<Self>) {
         self.client.event(Capabilities {
             self_id: self.id,
-            capabilities: POINTER | KEYBOARD,
+            capabilities: POINTER | KEYBOARD | TOUCH,
```

Comment:
We might want to only advertise touch if any touch devices are connected.

Reply:
It looks like the capabilities event is only sent to clients once at the start, shouldn't be sent every time a device is added/removed?

Reply:
Currently the capabilities are always `POINTER | KEYBOARD` so there is no need for this. But if the capabilities can actually change then this event should be sent whenever the capabilities change.

Reply:
I added a check for any connected touch devices.